packages/wallets/src/components/WalletsAddMoreCardBanner/WalletsAddMoreCardBanner.tsx

```diff
@@ -1,17 +1,95 @@
+    const { isMobile } = useDevice();
+    const history = useHistory();
+    const modal = useModal();
+    const syncLocalStorageClientAccounts = useSyncLocalStorageClientAccounts();
+
+    const renderButtons = useCallback(
+        () => (
+            <div className='wallets-add-more__success-footer'>
+                <WalletButton color='black' onClick={() => modal.hide()} text='Maybe later' variant='outlined' />
+                <WalletButton onClick={() => history.push('wallets/cashier/deposit')} text='Deposit now' />
+            </div>
+        ),
+        [history, modal]
+    );
+
+    useEffect(() => {
+        if (data && isMutateSuccess) {
+            syncLocalStorageClientAccounts(data);
+            switchAccount(data?.client_id);
+        }
+    }, [data, isMutateSuccess, switchAccount, syncLocalStorageClientAccounts]);
+
+    useEffect(() => {
+        if (status === 'error') {
+            modal.show(
+                <WalletError
+                    buttonText='Close'
+                    errorMessage={error.error.message}
+                    onClick={() => modal.hide()}
+                    type='modal'
+                />
+            );
+        } else if (status === 'success') {
+            modal.show(
+                <React.Fragment>
+                    {isMobile && (
+                        <ModalStepWrapper renderFooter={renderButtons} title=''>
+                            <Success
+                                description='Make a deposit into your new Wallet.'
+                                displayBalance={`0 ${data?.currency}`}
+                                marketType='all'
+                                platform='mt5'
+                                renderButton={() => null}
+                                title={`Your ${
+                                    data?.currency
+                                } wallet (${data?.landing_company_shortcode?.toUpperCase()}) is ready`}
+                            />
+                        </ModalStepWrapper>
+                    )}
+                    {!isMobile && (
+                        <ModalWrapper>
+                            <Success
+                                description='Make a deposit into your new Wallet.'
+                                displayBalance={`0 ${data?.currency}`}
+                                marketType='all'
+                                platform='mt5'
+                                renderButton={renderButtons}
+                                title={`Your ${
+                                    data?.currency
+                                } wallet (${data?.landing_company_shortcode?.toUpperCase()}) is ready`}
+                            />
+                        </ModalWrapper>
+                    )}
+                </React.Fragment>
+            );
+        }
+    }, [status]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
The dependencies should be updated right ?

Reply:
will check on it